AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -206,10 +212,10 @@ - (void)flushQueue {
 
                             // Notify delegates.
                             [self
-                                enumerateDelegatesForSelector:@selector(channel:didFailSendingLog:withError:)
+                                enumerateDelegatesForSelector:@selector(didFailSendingLog:withError:)
                                                     withBlock:^(id<MSChannelDelegate> delegate) {
                                                       for (id<MSLog> aLog in logArray) {
-                                                        [delegate channel:self didFailSendingLog:aLog withError:error];
+                                                        [delegate didFailSendingLog:aLog withError:error];
```

Comment:
Not a blocker but maybe something we want to discuss: It's common in ObjC for delegate methods to pass on a reference to the calling object (e.g. previously the `channel` parameter), e.g. for a discussion see  [here](https://stackoverflow.com/questions/37102504/proper-naming-convention-for-a-delegate-method-with-no-arguments-except-the-dele).

Reply:
I have no problem with re-adding it. The only reason I removed it is that now those who subscribe are likelier to have a reference to the channel already, but - that isn't always the case, and if it's convention to pass the sender, then it should definitely be an argument

Reply:
Then please re-add it